packages/apollos-church-api/config.yml

```diff
@@ -50,6 +52,7 @@ ANALYTICS:
   # on the accompanying service
   SEGMENT_KEY:
   GA_ID:
+  USE_ROCK:
```

Comment:
should this pull from our `.env` file? we should match however core does it

Reply:
@redreceipt Yep, the one that came in from core in the update just said `USE_ROCK: true`. I'll make it `false`.